src/components/molecules/Settings/Avatar/index.tsx

```diff
@@ -1,14 +1,17 @@
 import { styled } from "@reearth/theme";
 
-const Avatar = styled.div<{ size: number; avatar?: string }>`
+const Avatar = styled.div<{ size: number; avatar?: string; color?: string; radius?: number }>`
   display: flex;
   align-items: center;
   justify-content: center;
+  flex-direction: column;
+  padding: 0px;
+  position: relative;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
   min-width: ${({ size }) => size}px;
   min-height: ${({ size }) => size}px;
-  font-size: 24px;
-  border-radius: 50%;
-  background: ${({ avatar }) => (avatar ? `url(${avatar});` : "#888888")};
+  border-radius: ${({ radius }) => radius}%;
+  background: ${({ avatar, color }) => (avatar ? `url(${avatar});` : color)};
 `;
```

Comment:
This should be made into a React component and moved into the atoms directory since it is simple and reused in multiple places.
Also, a couple additional things:
-  This should be useable on its own without more styling since in @lavalse's design the avatar looks the same (except for the `size`).  This means the Text component should be used inside this component (rather than how you coded it now, which you pass the Text component as a child everywhere this component is used).
- Personally I feel like this component should take a username and do the processing itself, rather than doing `avatar?.charAt(0).toUpperCase()` everywhere that this component is used.

This means the `Avatar` atom component's props should be: `size`, `username` and for future use `color`. A `radius` prop isn't needed bc we don't need a square or softened cornered square avatar.

Reply:
If the Avatar components are USUALLY one size, and only sometimes another size, set a default size when destructuring your props in this component!